adapters/taboola/taboola.go

```diff
@@ -35,9 +38,15 @@ func (a *adapter) MakeRequests(request *openrtb2.BidRequest, requestInfo *adapte
 		return nil, []error{err}
 	}
 
+	parsedUrl, err := url.Parse(a.externalUrl)
+	if err != nil {
+		parsedUrl = nil
+	}
+	hostName := parsedUrl.Host
+
 	requestData := &adapters.RequestData{
 		Method: "POST",
-		Uri:    a.endpoint + "/" + taboolaRequest.Site.ID,
+		Uri:    a.endpoint + "/" + hostName + "/" + taboolaRequest.Site.ID,
```

Comment:
Can you please add a check to make sure that `hostName` is not an empty string before using it here? `ExternalUrl` is not required so a host can potentially set it to an empty string if they wanted to. 

Please also add a test for the empty external URL

Reply:
By default we want it to be an empty string. 
I added a check for this case and a unit test `TestEmptyExternalUrl` in `taboola_test.go`

Reply:
@ahmadlob So to confirm, your bidder service will accept a URL with empty path parameter like:
http://whatever.com//1234 ?

Reply:
@mansinahar  yes - it's valid